hack/golangci_lint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# check for golangci-lint in *.go files
+
+check=0
+
+test_golangci() {
+        golangci-lint run ./...
+        if [ $? -eq 0 ]; then
+                check=1
+        fi
+}	
+
+echo "Installing golangci-lint in bin"
+
+wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.31.0
```

Comment:
If I'm understanding this correctly, this installs it into the system's `/bin`, which we don't want to do.